src/components/Terminal/Terminal.tsx

```diff
@@ -107,6 +107,9 @@ class Terminal extends React.Component<TerminalProps, {}> {
 
   componentDidMount() {
     document.addEventListener('visibilitychange', this.handleVisibilityChange);
+    this.props.rootStore.uiStore.setPageVisibility(
+      document.visibilityState === 'hidden'
```

Comment:
I think we need to revert value of getPageVisibility function to return true if page is visible. Same logic for setPageVisibility.